cypress/e2e/shared/firstmile.tests.ts

```diff
@@ -0,0 +1,55 @@
+describe('First mile experience', () => {
+  beforeEach(() => {
+    cy.flush()
+    cy.signin()
+    cy.setFeatureFlags({firstMile: true})
+  })
+
+  it('should load the main home-page that has the language tiles', function() {
+    cy.getByTestID('home-page--header').should('exist')
+    cy.getByTestID('language-tiles--scrollbox').should('exist')
+  })
+
+  describe('Python', () => {
```

Comment:
let's add another describe block for the subway navigation. The things it should check:

- doing something like `cy.contains('Create a Token').click()` takes you to the tokens page
  - this is a case where clicking a nav link with the text `Create a Token` and then checking that the page it goes to has the text `Create a Token` is probably a good behavior to test, since from a product standpoint, we want the header text to match the link text
- clicking `cy.contains('Next')` on a page takes you to the next page, similar with `cy.contains('Previous')`
  - test that the navigation doesn't let you go past the last page and before the first page.